### PR TITLE
Update Readme.

### PR DESCRIPTION
**Pull request by zzl0**
Fix error in spark batch task example.

### CODE REVIEW

**Comment by zzl0** [comment]
@drcrallen pls review

**Comment by drcrallen** [comment]
@zzl0 missed this PR sorry about that. Checking it out.

File: README.md [hunk 1]
```diff
@@ -35,13 +35,10 @@ The following is an example spark batch task for the indexing service:
 
 ```json
 {
-    "dataFiles":["file:///Users/charlesallen/bin/wrk/lineitem.small.tbl"],
+    "paths":["/<your-druid-spark-batch-dir>/src/test/resources/lineitem.small.tbl"],
     "dataSchema": {
         "dataSource": "sparkTest",
         "granularitySpec": {
-            "intervals": [
```
Comment (drcrallen) [inline README.md]: I do not have a test suite for cases where intervals are not specified in granularity spec. Can you remove this change?

**Comment by zzl0** [comment]
Updated (and intervals field for granularitySpec)

PS, It is ugly to write `intervals` filed two times, may be `druid-spark-batch` should get the `intervals` value from granularitySpec, and remove the outer level `intervals` field.

**Comment by drcrallen** [comment]
@zzl0 Agreed. I'm actually not even sure it WOULDN'T work, but since I don't have unit tests or production experience running it that way I can't say one way or the other.

**Comment by drcrallen** [comment]
@zzl0 thanks for the update. Would you mind filling out the CLA at http://druid.io/community/cla.html (it is a general Metamarkets open source contributor CLA)?

**Comment by zzl0** [comment]
@drcrallen done :)

**Comment by drcrallen** [comment]
Sweet thanks for the contrib!

**Pull request merged**